### DISCUSSION TITLE
data importer creates duplicates / data importer does not update pending transactions if they are done

### DISCUSSION THREAD

**Discussion by chris1284** [firefly-iii/firefly-iii#9356] [Firefly III]
hi,
if i import data from my account, there are announced/pending transactions with no real details. 
only a cryptic description like "MO 560xxxxx xxxxxxxxxxxxC15" and a value "8,99€". 
they have the label [pending] automatically set from firefly iii. 
if they are book at next import, they have all infos in cleartext like readable destination account, description and so on. 

why is the data importer not updating this information? i have tried to set the import date to "today" to avoid import of "MO xxxxx...xx" transaction import, but if the booking date in the past of today and the real value/booking date is tomorrow, the are imported. for example booking date 11.10.2024, import today (13.10.2024) and the real value/booking date is monday 14.10.2024.  if i do an import on 14.10.2024 the transaction is there two times. one with description "MO 560xxxxx xxxxxxxxxxxxC15" and a value "8,99€" and another with description
2024-10-11T08:40 Debitk.15 2026-12" and a value "8,99€". 

Is there a way to exclude pending transactions? 

thx

**Comment by JC5** [comment]
The best way to stop this from happening is to make a rule that deletes transactions with the "pending" tag.
